crates/red_knot_python_semantic/src/types.rs

```diff
@@ -278,6 +280,11 @@ impl<'db> Type<'db> {
                 Type::Unknown
             }
             Type::BooleanLiteral(_) => Type::Unknown,
+            Type::StringLiteral(_) => {
+                // TODO fix this comment with whatever it is we should say we
+                // want to do in the future.
```

Comment:
I expect this is probably something in the space of what we say for `BytesLiteral`?